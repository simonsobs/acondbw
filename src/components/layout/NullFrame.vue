<template>
  <div></div>
</template>

<script>
export default {
  name: "NullFrame",
  data: () => ({}),
};
</script>
