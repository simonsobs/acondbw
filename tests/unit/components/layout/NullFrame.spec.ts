import Vue from "vue";
import Vuetify from "vuetify";
import { shallowMount, createLocalVue } from "@vue/test-utils";

import NullFrame from "@/components/layout/NullFrame.vue";

Vue.use(Vuetify);

describe("Frame.vue", () => {
  let localVue: ReturnType<typeof createLocalVue>;
  let vuetify: Vuetify;
  let wrapper: ReturnType<typeof shallowMount>;

  beforeEach(() => {
    localVue = createLocalVue();
    vuetify = new Vuetify();
    wrapper = shallowMount(NullFrame, {
      localVue,
      vuetify,
    });
  });

  it("match snapshot", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
