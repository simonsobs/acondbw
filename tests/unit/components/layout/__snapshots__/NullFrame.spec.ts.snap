// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Frame.vue match snapshot 1`] = `<div></div>`;
